usecases/backup/zip.go

```diff
@@ -129,7 +129,15 @@ func (z *zip) WriteRegular(ctx context.Context, relPath string) (written int64,
 	absPath := filepath.Join(z.sourcePath, relPath)
 	info, err := os.Stat(absPath)
 	if err != nil {
-		return written, fmt.Errorf("stat: %w", err)
+		if os.IsNotExist(err) {
+			absPath = filepath.Join(z.sourcePath, entBackup.DeleteMarker+relPath)
+			info, err = os.Stat(absPath)
+			if err != nil {
```

Comment:
here you would like to check if the file still exists, yes?

Reply:
yep, added a comment to make it clearer